src/components/MapDisplay/MapDisplay.js

```diff
@@ -3,35 +3,103 @@ import React, { Component } from 'react'
 import './MapDisplay.css'
 
 const meterConversion = 1609.34;
+const CONSTANTS = {
+  tileURL: 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
+  attribution: 'Map data © <a href="https://openstreetmap.org">OpenStreetMap</a> contributors',
+  minZoom: 4,
+  maxZoom: 20,
+  defaultZoom: 10,
+  meterConversion: 1609.34
+};
+const styles = {
+  red: {
+    color: 'red',
+    fillColor: '#f03',
+    fillOpacity: 0.5
+  }
+};
 
 class MapDisplay extends Component {
```

Comment:
this file might be best as a hook and useEffect

Reply:
but that can be for another branch

Reply:
It was already written with the older componentDidMount style so I just added the update lifecycle method in the same way. I still need to read up more on using hooks and useEffect so I decided I'll just leave that for a future update.